HHS/Head-Start-TTADP
# [Prod] Resolve email notifications issue; handle Unix timestamp disclosure; reset RTR goals filter on changes; address warnings

## Description of change

TTAHUB-3311 Fix for several on-demand AR email notifications, including an approval notification
TTAHUB-3042 Ignore Unix timestamp disclosure alert
TTAHUB-2765 Reset the RTR goals filter when the filter changes.
TTAHUB-3110 Address warnings for the Similarity API
TTAHUB-3313 SSDI: standard-goal-activity-reports

## How to test

TTAHUB-2765 Go to the RTR goals page. Go to the second page. Modify or add a filter. You should reset back to page one.
TTAHUB-3313 https://ttahub.ohs.acf.hhs.gov/api/admin/ssdi/run-query?path=src/queries/standard-goal-activity-reports.sql&format=json

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3311
* https://jira.acf.gov/browse/TTAHUB-3042
* https://jira.acf.gov/browse/TTAHUB-2765
* https://jira.acf.gov/browse/TTAHUB-3110
* https://jira.acf.gov/browse/TTAHUB-3313


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Before merge to main

- [n/a] OHS demo complete

### Production Deploy


### After merge/deploy
